actioncable/test/subscription_adapter/redis_test.rb

```diff
@@ -14,3 +14,9 @@ def cable_config
     super.merge(driver: "hiredis")
   end
 end
+
+class RedisAdapterWithChannelPrefixTest < RedisAdapterTest
```

Comment:
I'm pretty sure this test would pass if it was added without any of the above implementation.

"It still works" is a good baseline, but we'll also need to prove we're achieving our aim: that two differently-configured adapters don't interfere with each other. (Ideally, we can make that a generic test for all adapters -- some will pass naturally, and others will be able to pull in the namespacing module.)